src/components/layout/MainOrgLayout.tsx

```diff
@@ -40,24 +40,26 @@ const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
     const tab3 = <Msg id="layout.org.tabs.campaigns"/>;
 
     return (
-        <DefaultLayout org={ orgQuery.data! }>
-            <OrgHeader org={ orgQuery.data! }/>
-            <Tabs
-                aria-label="Organization submenu"
-                onSelectionChange={ onSelectTab }
-                selectedKey={ currentTab }>
-                <Item key="home" title={ tab1 }>
-                    <Content>{ children }</Content>
-                </Item>
-                <Item key="events" title={ tab2 }>
-                    <Content>{ children }</Content>
-                </Item>
-                <Item key="campaigns" title={ tab3 }>
-                    <Content>{ children }</Content>
-                </Item>
-            </Tabs>
-        </DefaultLayout>
+        <DefaultOrgLayout orgId={ orgId }>
+            <>
```

Comment:
Is this fragment (`<>`) really necessary?

Reply:
I added this because only one child was expected. Maybe there's a better way?

Reply:
Why would you say that only one child was expected? Was there a warning or something?

It's entirely possible and normal in React to have multiple children of an element. In fact, by moving the children into a fragment (`<>` or `React.Fragment`) you still have multiple children, except that they're children of another element (the fragment).

If our layout components are complaining about adding multiple children, we need to change their expectations.

Reply:
Okay! Yes, the `DefaultOrgLayout` component expected one but got several children.